homeassistant/components/switchbot/strings.json

```diff
@@ -36,6 +36,27 @@
           "lock_auth": "SwitchBot account (recommended)",
           "lock_key": "Enter lock encryption key manually"
         }
+      },
+      "key": {
+        "description": "The {name} device requires encryption key, details on how to obtain it can be found in the documentation.",
+        "data": {
+          "key_id": "Key ID",
+          "encryption_key": "Encryption key"
+        }
+      },
+      "auth": {
+        "description": "Please provide your SwitchBot app username and password. This data won't be saved and only used to retrieve your device encryption key. Usernames and passwords are case sensitive.",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
+      "choose_method": {
+        "description": "A SwitchBot device can be set up in Home Assistant in two different ways.\n\nYou can enter the key id and encryption key yourself, or Home Assistant can import them from your SwitchBot account.",
+        "menu_options": {
+          "auth": "SwitchBot account (recommended)",
```

Comment:
Why do we recommend that?

Reply:
Because there is currently no open access to obtain the encryption key within our app, users do not know where to retrieve it from.